plugin-gradle/src/main/java/com/diffplug/gradle/spotless/TypescriptExtension.java

```diff
@@ -108,10 +139,26 @@ public PrettierConfig prettier() {
 		return prettierConfig;
 	}
 
+	@Override
+	public PrettierConfig prettier(String prettierVersion) {
+		PrettierConfig prettierConfig = new TypescriptPrettierConfig(prettierVersion);
+		addStep(prettierConfig.createStep());
+		return prettierConfig;
+	}
+
 	/**
 	 * Overrides the parser to be set to typescript, no matter what the user's config says.
 	 */
 	public class TypescriptPrettierConfig extends PrettierConfig {
+
+		TypescriptPrettierConfig() {
+			super();
```

Comment:
When possible, I think it's safer to have only one constructor that "does something", and everything else should delegate to that.  So in this case, I think line ought to be `this(DEFAULT_PRETTIER_VERSION)` rather than `super()`.